lib/utils/file.js

```diff
@@ -0,0 +1,17 @@
+// Based on https://github.com/kevva/executable
+// Since this module doesn't expose the function to check stat mode only,
+// his logic is pasted here.
+// Opened an issue and a pull request about it,
+// to maybe switch to module in the future:
+// Issue: https://github.com/kevva/executable/issues/9
+// PR: https://github.com/kevva/executable/pull/10
+
+export function isExecutable (fileStat) {
+  if (process.platform === 'win32') return true;
```

Comment:
I agree it's odd, it's a design decision at this point. 

Starting from the idea that when "executable" file is opened, a new line (`\n`) will be appended to the path, we have to define what an executable file is. 

Since windows support is not still defined, I decided to keep the original implementation of "executable" npm module and return a fixed value. Now we alway have possible false positive results, but we can change it to have always false negative results.

In a *nix environment an executable file is a file with a specific mode for a specific user/group/admin; but on windows it's not so clear.
It could be a file with an extension that appears inside [PATHEXT env variable](http://environmentvariables.org/PathExt), but I don't know if the shell we'll use on windows will be capable of execute .EXE, .BAT, etc.

However, maybe we should avoid to use "executable" external module and keep this logic inside our code.

@rauchg @albinekb What do you think about that?